backend/src/schemas/entities/interrupt.py

```
@@ -0,0 +1,226 @@
+        description="Optional reason for the decision (useful for reject/respond)",
+    )
+    nonce: Optional[str] = Field(
+        default=None, description="One-time-use token for replay protection"
+    )
+
+    @field_validator("edited_args")
+    @classmethod
+    def validate_edited_args_required_for_edit(cls, v, info):
+        """Ensure edited_args is provided when action is 'edit'."""
+        if info.data.get("action") == DecisionType.EDIT and not v:
+            raise ValueError("edited_args is required when action is 'edit'")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)